site/docs/capabilities/fallback.md

```diff
@@ -0,0 +1,113 @@
+# Provider Fallback
+
+Envoy AI Gateway supports provider fallback to ensure high availability and reliability for AI/LLM workloads. With fallback, you can configure multiple upstream providers for a single route, so that if the primary provider fails (due to network errors, 5xx responses, or other health check failures), traffic is automatically routed to a healthy fallback provider.
+
+## How Fallback Works
+
+- **Primary and Fallback Backends:** You can specify a prioritized list of backends in your `AIGatewayRoute` using `backendRefs`. The first backend is treated as primary, and subsequent backends are considered fallbacks.
+- **Health Checks:** Fallback is triggered based on passive health checks and retry policies, which can be configured using the [`BackendTrafficPolicy`](https://gateway.envoyproxy.io/contributions/design/backend-traffic-policy/) API.
```

Comment:
Worth to mention that the the HttpRouter is sharing the same name with the AIGatewayRoute.

Reply:
add a note on the HTTPRoute name reference